templates/Dockerfile.erb

```diff
@@ -19,7 +19,7 @@ ENV PATH /fluentd/vendor/bundle/ruby/2.3.0/bin:$PATH
 ENV GEM_PATH /fluentd/vendor/bundle/ruby/2.3.0
 ENV GEM_HOME /fluentd/vendor/bundle/ruby/2.3.0
 # skip runtime bundler installation
-ENV FLUENTD_DISABLE_BUNDLER_INJECTION 1
```

Comment:
Nit: We could just skip adding the environment variable all together where required ...

like
```ruby
<% unless requires_git %>
# skip runtime bundler installation
ENV FLUENTD_DISABLE_BUNDLER_INJECTION 1
<% end %>
```